src/views/PatronsPreRegistrationListContainer/hooks/useCreateNewUser.test.js

```diff
@@ -0,0 +1,96 @@
+import {
+  QueryClient,
+  QueryClientProvider,
+} from 'react-query';
+import { useHistory } from 'react-router-dom';
+
+import { renderHook, waitFor } from '@folio/jest-config-stripes/testing-library/react';
+import preRegistrationRecords from 'fixtures/preRegistrationRecords';
+import { useOkapiKy, useCallout } from '@folio/stripes/core';
+
+import { PATRON_PREREGISTRATIONS_API } from '../../../constants';
+import useCreateNewUser from './useCreateNewUser';
+
+jest.mock('react-router-dom', () => ({
+  useHistory: jest.fn(),
+}));
+jest.mock('@folio/stripes/core', () => ({
```

Comment:
`@folio/stripes/core` already mocked in the jest setup: `test/jest/__mock__/stripesCore.mock.js`

Reply:
picking up that mock from configuration is leading to some issue. In the interest of time, I followed this way.